src/blocks/shared/functions.scss

```diff
@@ -0,0 +1,12 @@
+@use "sass:math";
+@import "./variables";
+
+@function stripUnit($value) {
+
+	@return math.div($value, $value * 0 + 1);
+}
+
+@function to_rem($pxValue) {
+
```

Comment:
These empty lines are due to our linting specs .... should we change it?

Reply:
Here is the error:

```
src/blocks/shared/functions.scss
 5:2  ✖  Expected empty line before at-rule  at-rule-empty-line-before
 9:2  ✖  Expected empty line before at-rule  at-rule-empty-line-before
 ```

Reply:
Let's leave this as is for now. We may revisit to fix it if it becomes an annoyance.